layers/meta-balena-topic/recipes-core/images/balena-image.bbappend

```diff
@@ -33,3 +33,9 @@ ROOTFS_POSTPROCESS_COMMAND += "topicbalena_rootfs_postprocess; "
 # We need to increase the rootfs size to make space for additional drivers in ROOTFS. 
 # BalenaOS image has a 700MB max size defined in 'image_types_balena.bbclass'
 IMAGE_ROOTFS_SIZE="409600"
+
```

Comment:
What is this needed for? Isn't `/etc/os-release` enough to provide with the info you need?

Reply:
os-revision has a 'nice' version of BalenaOS, but a commit hash is the absolute truth.
I suppose this only helps while developing on a branch, where the os-revision stays fixed. On the 'master'/'production' BalenaOS revision does get stepped quite regularly.
Doesn't hurt to have it here, but I can easily remove it as well.